bugbug/bug_features.py

```diff
@@ -481,6 +481,22 @@ def __call__(self, bugs, **kwargs):
         return bugs[0]["target_milestone"] == bugs[1]["target_milestone"]
 
 
+class is_first_affected_same(couple_bug_feature):
+    def __call__(self, bugs, **kwargs):
```

Comment:
I suggest moving the code at https://github.com/mozilla/bugbug/blob/7f5f43d4c63337e2a475998bd1d8003dca752b8c/bugbug/bug_features.py#L265-L288 in a helper function and reusing it here.